deployments/fpga_admissionwebhook/webhook/manifests.yaml

```diff
@@ -12,6 +12,7 @@ webhooks:
       name: webhook-service
       namespace: system
       path: /pods
+  failurePolicy: Ignore
```

Comment:
Does this mean that we ignore webhook service failures?

Reply:
Yes, otherwise the webhook itself cannot start (because its pod doesn't run yet when apiserver tries to mutate it).